src/index.ts

```diff
@@ -15,7 +15,7 @@ const argv = yargs
   })
   .option('page-size', {
     describe: 'The number of orders to fetch per page',
-    default: 50,
+    default: 100,
```

Comment:
Any particular reason for the bump here?

Reply:
performance boost in the way we query the orderbook